src/vm/runners/cairo_runner.zig

```diff
@@ -1253,3 +1316,47 @@ test "CairoRunner: getPermRangeCheckLimits with null range limit" {
         try cairo_runner.getPermRangeCheckLimits(std.testing.allocator),
     );
 }
+
+test "CairoRunner: initBuiltins missing builtins allow missing" {
+    var program = try Program.initDefault(std.testing.allocator, true);
+    try program.builtins.appendSlice(&.{ .output, .ecdsa });
+    // Initialize a CairoRunner with an empty program, "plain" layout, and instructions.
+    var cairo_runner = try CairoRunner.init(
+        std.testing.allocator,
+        program,
+        "plain",
+        ArrayList(MaybeRelocatable).init(std.testing.allocator),
+        try CairoVM.init(
+            std.testing.allocator,
+            .{},
+        ),
+        false,
+    );
+
+    // Defer the deinitialization of the CairoRunner to ensure cleanup.
+    defer cairo_runner.deinit(std.testing.allocator);
+
+    try cairo_runner.initBuiltins(true);
```

Comment:
Here can we also add an assertion in order to test that the builtins are properly initialized inside the vm of the Cairo runner?